guide/src/tutorials/draw/drawing-images.md

```diff
@@ -0,0 +1,188 @@
+
+You can also find this file, and other useful examples, in the [examples](https://github.com/nannou-org/nannou/tree/master/examples) directory of the nannou source repository.
+
+To begin with, we will need a nannou project file to work with. Copy the following into new file:
+
+```rust,no_run
+# #![allow(unreachable_code, unused_variables, dead_code)]
+use nannou::prelude::*;
+
+struct Model {}
+
+fn main() {
+  nannou::app(model).run()
+}
+
+fn model(_app: &App) -> Model {
+  Model {}
+}
+
+fn view(_app: &App, _model: &Model, _frame: Frame) {
+}
+```
+
+Next, we will also need to bootstrap an `assets` directory with some images at the root of our project. I recommend copying the assets directory from the `nannou` repo itself to get started.
+
+```bash
+git clone https://github.com/nannou-org/nannou.git
+cp -r nannou/assets ./assets
+rm -rf nannou
+```
+
+Once you have a fleshed-out `assets` directory at the root of your project, you'll be ready to return to your `main.rs` file for the following steps.
+
+## Creating a Window
+
+We need somewhere to display the image, so first, let's create a window:
+
+```rust,no_run
+# #![allow(unreachable_code, unused_variables, dead_code)]
+# use nannou::prelude::*;
+# struct Model {}
+# fn main() {
+#   nannou::app(model).run()
+# }
+fn model(app: &App) -> Model {
+  // Create a new window!
+  app.new_window().size(512, 512).view(view).build().unwrap();
+  Model {}
+}
+# fn view(_app: &App, _model: &Model, _frame: Frame) {
+# }
+```
+
+If you `cargo run` your app, you'll see an empty green window.
+
+## Setting up a Texture
+
+Now, at the top of your `main.rs` file, add a [WGPU Texture](https://docs.rs/nannou/latest/nannou/wgpu/struct.Texture.html)-type field named `texture` to the `Model` struct.
+
+```rust,no_run
+# #![allow(unreachable_code, unused_variables, dead_code)]
+# use nannou::prelude::*;
+struct Model {
+  texture: wgpu::Texture,
+}
+# fn main() {
+#   nannou::app(model).run()
+# }
+# fn model(app: &App) -> Model {
+#   // Create a new window!
+#   app.new_window().size(512, 512).view(view).build().unwrap();
+#   let texture: wgpu::Texture = unimplemented!();
+#   Model { texture }
+# }
+# fn view(_app: &App, _model: &Model, _frame: Frame) {
+# }
+```
+
+Next, we'll need to create a GPU texture to initialize the struct with. We can accomplish this by loading a texture from an image file after we create the window in our `model` function.
+
+```rust,no_run
+# #![allow(unreachable_code, unused_variables, dead_code)]
+# use nannou::prelude::*;
+# struct Model {
+#   texture: wgpu::Texture,
+# }
+# fn main() {
+#   nannou::app(model).run()
+# }
+fn model(app: &App) -> Model {
+  // Create a new window!
+  app.new_window().size(512, 512).view(view).build().unwrap();
+  // Load the image from disk and upload it to a GPU texture.
+  let assets = app.assets_path().unwrap();
```

Comment:
It might be worthwhile to link or point out the `assets_path` function in case the reader did not place the assets in the desired location.

Reply:
I just added a sentence about it.